src/main/java/uk/gov/pay/adminusers/persistence/dao/UserDao.java

```diff
@@ -31,21 +31,21 @@ public UserDao(Provider<EntityManager> entityManager) {
 
     public Optional<UserEntity> findByUsername(String username) {
         String query = "SELECT u FROM UserEntity u " +
-                "WHERE u.username = :username";
+                "WHERE LOWER(u.username) = :username";
```

Comment:
might be better to use PG's `lower()` function on both strings to avoid the possibility that e.g. unicode chars get treated differently by the different implementations